src/main/java/wooteco/subway/web/member/interceptor/BearerAuthInterceptor.java

```diff
@@ -1,48 +1,59 @@
 package wooteco.subway.web.member.interceptor;
 
+import static wooteco.subway.web.exception.UnauthorizedException.*;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
 import org.springframework.stereotype.Component;
 import org.springframework.web.servlet.HandlerInterceptor;
 import org.springframework.web.servlet.ModelAndView;
+
 import wooteco.subway.infra.JwtTokenProvider;
+import wooteco.subway.web.exception.UnauthorizedException;
 import wooteco.subway.web.member.AuthorizationExtractor;
 
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
 @Component
 public class BearerAuthInterceptor implements HandlerInterceptor {
+    public static final String BEARER = "bearer";
+
     private AuthorizationExtractor authExtractor;
     private JwtTokenProvider jwtTokenProvider;
 
-    public BearerAuthInterceptor(AuthorizationExtractor authExtractor, JwtTokenProvider jwtTokenProvider) {
+    public BearerAuthInterceptor(AuthorizationExtractor authExtractor,
+        JwtTokenProvider jwtTokenProvider) {
         this.authExtractor = authExtractor;
         this.jwtTokenProvider = jwtTokenProvider;
     }
 
     @Override
     public boolean preHandle(HttpServletRequest request,
-                             HttpServletResponse response, Object handler) {
-        // TODO: Authorization 헤더를 통해 Bearer 값을 추출 (authExtractor.extract() 메서드 활용)
-
-        // TODO: 추출한 토큰값의 유효성 검사 (jwtTokenProvider.validateToken() 메서드 활용)
-
-        // TODO: 추출한 토큰값에서 email 정보 추출 (jwtTokenProvider.getSubject() 메서드 활용)
-        String email = "";
-
+        HttpServletResponse response, Object handler) {
+        // 회원가입 시 토큰 검증 불필요
+        if (request.getMethod().equals("POST") && request.getRequestURI().equals("/members")) {
+            return true;
+        }
+        String credential = authExtractor.extract(request, BEARER);
+        if (!jwtTokenProvider.validateToken(credential)) {
```

Comment:
```suggestion
        if (jwtTokenProvider.isInvalidToken(credential)) {
```

부정문은 가독성을 해칠 수 있습니다.
Objects.isNull과 Objects.nonNull이 존재하는 이유에 대해 고민해보아요.

Reply:
좋은 리뷰 감사합니다! 수정완료하였습니다😁